src/calculateRoundedRating/calculateRoundedRating.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/**
+ * Implement calculateRoundedRating function:
+ *
+ * Function takes an exact rating of the product - the float number between 1
+ * and 5 including (range `[1, 5]`) and returns the rounded rating to closest
+ * integer (1, 2, 3, 4 or 5).
+ *
+ * calculateRoundedRating(3.66666) === 4
```

Comment:
'round' is a verb, so you can name it 'roundRating'. No need to rename, this is fine.